AppCreationScripts/apps.json

```diff
@@ -48,7 +48,9 @@
 		      "replaceTokens": {
```

Comment:
I think in Archie's instructions said not include the root folder in settingFile and only include relative path. Is this still correct?

Reply:
I see we have another PR from archie #10 solving that. It just needs to be merged